src/mpcCoreKit.ts

```diff
@@ -1543,7 +1592,8 @@ export class Web3AuthMPCCoreKit implements ICoreKit, IMPCContext {
       additionalMetadata = {};
     }
 
-    const { tssIndex } = await this.getTssShare(factorKey);
+    const firstKeyType = this.getSupportedCurveKeyTypes()[0];
+    const { tssIndex } = await this.getTssShare({ keyType: firstKeyType, factorkey: factorKey });
```

Comment:
Isn't this wrong? What if you want to get the `ed25519` share but first key type is `secp256k1`? Same applies to several other instances of first key type usage.

Reply:
I are trying to get tssIndex from getTssShare or getFactorPubs from getTssData.
both curves should have the same tssIndex (for same factorKey) and factorPub 
Using either curve's keyType will get the same value.

using the first as instance might configured with single curve keytype only